packages/data-service/src/index-detail-helper.ts

```diff
@@ -69,7 +69,7 @@ export function getIndexProperties(
     properties.push('partial');
   }
 
-  if (index.extra.expireAfterSeconds > -1) {
+  if (Number(index.extra.expireAfterSeconds) > -1) {
```

Comment:
That was the only new failure I was getting locally when compiling after update

Reply:
They are not explicitly documented and not provided by driver, so it would take some work to come up with a correct type here, yeah...